### DISCUSSION TITLE
My plug has two switches that can control and report data separately. How can I display the two switches and their corresponding power, accumulated power and other information in two blocks on the page?

### DISCUSSION THREAD

**Discussion by hwzolin** [zigpy/zha-device-handlers#3622] [Q&A]
Hello everyone, our company has launched a new PLUG with two slots that can control the opening and closing of the two slots respectively and can report the power and accumulated power generated by the electrical appliances on both sides during use.

![screenshot1](https://github.com/user-attachments/assets/c349b478-9db5-4f13-99ef-29342f51edba)

However, as shown in the attached picture, after I added this PLUG to ZHA, all the function fields were merged together, and the corresponding relationship was not clear. Now I want to separate the switch, power, and accumulated power into an independent block according to the slot.

How can I modify it?

**Comment by TheJulianJES** [comment]
You can't modify this at the moment. In the future, we might be able to add the endpoint id in the name if there are duplicate entities.

**Reply by hwzolin** [reply]
Excuse me, how do I add the endpoint id?
